main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -1493,8 +1493,10 @@ object Namer extends Phase[WeededAst.Program, NamedAst.Root] {
     * Returns the class reflection object for the given `className`.
     */
   // TODO: Deprecated should be moved to resolver.
-  private def lookupClass(className: String, loc: SourceLocation): Validation[Class[_], NameError] = try {
-    Class.forName(className).toSuccess
+  private def lookupClass(className: String, loc: SourceLocation)(implicit flix: Flix): Validation[Class[_], NameError] = try {
```

Comment:
Remind me again why we have lookupClass in both Namer and Resolver?

Reply:
This is only used for catch cases. I think we need a bit of refactoring to avoid having to do this here.

Reply:
OK, can you create a ticket?

Reply:
https://github.com/flix/flix/issues/2616